tutorial/nemo/nemo_example1.ipynb

```diff
@@ -0,0 +1,316 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PSyclone tutorial: NEMO API Example 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example introduces PSyclone's NEMO API and shows how Fortran code is translated into PSyclone's language independent internal representation (the PSyIR) and then translated back into code."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First let's specify a simple Fortran code in a Python string. PSyclone typically reads codes from files but it is easier to demonstrate what is happening using a string."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the code consists of nested loops containing accesses to multi-dimensional arrays. This code structure is typical of finite difference and finite volume codes running on regular meshes and is what the NEMO API expects. If code is written in a different way e.g. with indirection to support irregular meshes then the NEMO API will not be useful and should not be used."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Also notice that the outer loops have indices `jk` with bounds from `1` to `jpk`, the middle loops have indices `jj` with bounds from `1` to `jpj` and the inner loops have indices `ji` with bounds from `1` to `jpi`. The NEMO coding rules enforce this naming convention throughout the code where `ji` corresponds to iterating over longitude, `jj` corresponds to iterating over latitude and `jk` corresponds to iterating over levels. PSyclone uses this naming convention to determine the type of loop. If a different naming scheme is used in a different code then PSyclone can be changed to use these instead. If you are interested you can find the mappings used in the PSyclone config file (which is pointed at by the `PSYCLONE_CONFIG` environment variable)."
+   ]
```

Comment:
PSYCLONE_CONFIG is not normally set. Are you OK if I replace this with a link to the appropriate part of the UG?

Reply:
Ah. Yes sure. I just thought they might like to have a look for the file and see the content.

Reply:
Yes sure. I just thought they might want to see the actual file.

Reply:
I've put a link to the file in the github repo - finding where the "actual file" is installed is tricky so I thought this would be simpler.

Reply:
Yes that's a good solution